src/Generator/ModelGenerator.php

```diff
@@ -56,7 +56,11 @@ public function generate(): void
                 $schema['type'] = 'object';
             }
 
-            $schemaType = $schema['type'];
+            $schemaType = $schema['type'] ?? null;
+
+            if (empty($schemaType)) {
+                throw new Exception('No type defined for schema: ' . $schemaName);
```

Comment:
I don't know about this, it seems to be valid in both json schema (which swagger/OA inherits from) and in the OA spec to NOT have a type. We should probably default to `object` if it's not specified.

Reply:
I really don't want to default in case of a type. If it's not defined it should fail and let the developer know about the issue.
I really don't get why would someone define an API without the type of the fields defined properly